### PR TITLE
fix: use three-vrm-animation

### PR DESCRIPTION
**Pull request by yue4u**
Replace vrma implementation with three-vrm-animation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +26,6 @@
     "next-auth": "3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-three-fiber": "^6.0.13",
```
Comment (yue4u) [inline package.json:28]: We have @react-three/fiber so this is no longer needed.

File: pages/vrm.tsx [hunk 2]
```diff
@@ -83,23 +82,30 @@ const Avatar = ({ vrm }: { vrm: VRM }) => {
     }
   });
 
+  const vrmaContainer = useLoader(GLTFLoader, '/idle_loop.vrma', (loader) => {
+    loader.register((parser) => {
+      return new VRMAnimationLoaderPlugin(parser);
+    });
+  });
+
```
Comment (yue4u) [inline pages/vrm.tsx:90]: https://github.com/yue4u/libvrm/blob/main/packages/use-vrm/src/use-vrm.ts#L84-L100

File: package.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
     "@charcoal-ui/react": "^3.1.0",
     "@charcoal-ui/styled": "^3.1.1",
     "@gltf-transform/core": "^3.4.8",
-    "@pixiv/three-vrm": "^1.0.10",
+    "@pixiv/three-vrm": "^3.1.4",
```
Comment (yue4u) [inline package.json:16]: - https://github.com/pixiv/three-vrm/releases/tag/v2.0.0
- https://github.com/pixiv/three-vrm/releases/tag/v3.0.0

**Comment by kajiina** [comment]
LGTM!

**Pull request merged**